src/components/TablePreview/TablePreview.tsx

```diff
@@ -38,17 +38,17 @@ const TablePreview: FC<Props> = ({}) => {
                             <img src={imgUrl} height="60"></img>
                         </td>
                         <td style={name.style}>{name.textValue}</td>
-                        <td>&nbsp;</td>
+                        <td style={{ width: "16px" }}></td>
                         <td style={phone.style}>{phone.textValue}</td>
```

Comment:
do we need a whole new table data tag <td></td> to change the width? could it be done within the table data tag that wrappes {phone.textValue} 🤔  maybe something like: `<td style={{ ...phone.style, width: '16px' }}>{phone.textValue}</td> `. Here the spread operator is used to merge the foo.style object with the css width property!
And only being applied to the table data cells that we would like to have the change in our HTML table 😸 without modding the table structure itself, thoughts @evargast ?